explainer.md

```diff
@@ -107,7 +112,7 @@ These programs must currently contend for the global shared resources, specifica
 
 Attempting to solve these problems with existing DOM APIs will require to implement an asynchronous communication protocol, which is often a deal-breaker for many use cases. It usually just adds complexity for cases where a same-process Realm is sufficient. It's also very important that values can be immediately shared. Other communications require data to be serialized before it's sent back and forth.
 
-__The primary goal of this proposal is to provide a proper mechanism to control the execution of a program, providing a new global object, a new set of intrinsics, no default access to the incubator realm's global object, a separate module graph and synchronous communication with the incubator realm__.
+__The primary goal of this proposal is to provide a proper mechanism to control the execution of a program, providing a new global object, a new set of intrinsics, no immediate access to objects cross-realms, a separate module graph and synchronous communication between both realms, while not forcing fingerprints in the constructed realm__.
```

Comment:
```suggestion
__The primary goal of this proposal is to provide a proper mechanism to control the execution of a program, providing a new global object, a new set of intrinsics, no access to objects cross-realms, a separate module graph and synchronous communication between both realms, while not forcing fingerprints in the constructed realm__.
```

Reply:
I also continue having a hard time understanding the finderprints issues with the previous proposal. can you elaborate more?

Reply:
I'm skeptical that this proposal changes the fingerprinting surface. I think that it'd be best to remove this claim.

Reply:
> I also continue having a hard time understanding the finderprints issues with the previous proposal. can you elaborate more?

That's just about identifying identity from other realms through objects that would then be accessed by `globalThis`. Of course any framework would handle that, and that's not any new complication (iframe allows that already).

I'll remove this part.

Reply:
7b88d54